jsonrpc-specs/src/methods/discv5.json

```diff
@@ -0,0 +1,567 @@
+					"enr": {
+						"title": "nodeENR",
+						"description": "URL-safe base64 encoded \\\"text\\\" version of the ENR prefixed by \\\"enr:\\\".\"",
+						"$ref": "#/components/schemas/Enr"
+					},
+					"nodeId": {
+						"title": "nodeId",
+						"description": "Hex encoded `NodeId` of an ENR (a 32 byte identifier).",
+						"$ref": "#/components/schemas/bytes32"
+					}
+				}
+			}
+		}
+	},
+	{
+		"name": "discv5_updateNodeInfo",
+		"summary": "Add, update, or remove a key-value pair from the local node record",
+		"params": [
+			{
+				"name": "Key/value pairs",
+				"required": true,
+				"schema": {
+					"title": "",
+					"type": "array",
+					"items": { "$ref": "#/components/schemas/KVpairs" }
+			}
+			}
+		],
+		"result": {
+			"name": "nodeInfoResult",
+			"description": "Local node information",
+			"required": true,
+			"schema": {
+				"title": "nodeInfoResults",
+				"description": "ENR and NodeId of the local peer",
+				"type": "object",
+				"required": [
+					"enr",
+					"nodeId"
+				],
+				"properties": {
+					"enr": {
+						"title": "nodeENR",
+						"description": "URL-safe base64 encoded \\\"text\\\" version of the ENR prefixed by \\\"enr:\\\".\"",
+						"$ref": "#/components/schemas/Enr"
+					},
+					"nodeId": {
+						"title": "nodeId",
+						"description": "Hex encoded `NodeId` of an ENR (a 32 byte identifier).",
+						"$ref": "#/components/schemas/bytes32"
+					}
+				}
+			}
+		}
+	},
+	{
+		"name": "discv5_routingTableInfo",
+		"summary": "Returns meta information about discv5 routing table.",
+		"params": [],
+		"result": {
+			"name": "routingTableInfoResult",
+			"description": "Discv5 routing table information",
+			"required": true,
+			"schema": {
+				"title": "routingTableInfoResults",
+				"description": "Routing table details",
+				"type": "object",
+				"required": [
+					"localKey",
+					"buckets"
+				],
+				"properties": {
+					"localKey": {
+						"title": "nodeId",
+						"description": "The key identifying the local peer that owns the routing table.",
+						"$ref": "#/components/schemas/bytes32"
+					},
+					"buckets":  {
+						"title": "kBucketsTable",
+						"description": "Represents a Kademlia routing table.",
+						"$ref": "#/components/schemas/kBuckets"
+					}
+				}
+			}
+		}
+	},
+	{
+		"name": "discv5_setEnr",
+		"summary": "Write an ethereum node record to the database.",
+		"params": [
+			{
+				"name": "enr",
+				"required": true,
+				"schema": {
+					"title": "Ethereum node record",
+					"$ref": "#/components/schemas/Enr"
+				}
+			}
+		],
+		"result": {
+			"name": "setEnrResult",
+			"description": "Returns ethereum node record if the node record has been successfully saved.",
+			"schema": {
+				"title": "Ethereum node record",
+				"$ref": "#/components/schemas/Enr"
+			}
+		}
+	},
+	{
+		"name": "discv5_getEnr",
+		"summary": "Fetch the latest ENR associated with the given node ID",
+		"params": [
+			{
+				"name": "node_id",
```

Comment:
Here we use snake case but in `discv5_nodeInfo` we use camel case. We might want to settle on one of the two.

Reply:
Hm, for me it makes sense to use `nodeId`, it is more json-ish according to [Google JSON Style Guide](https://google.github.io/styleguide/jsoncstyleguide.xml) but Ethereum JSON-RPC [specs](https://github.com/ethereum/execution-apis/blob/eab7edd4bcb144cee75233054bcc2016847aeaa9/src/eth/block.json#L7) use spacing between the words. Do you have any preferences?

Reply:
Changing this to camel case for now.

Reply:
I'm fine with that. No real preference between snake or camel case. Spacing is strange.